<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>740</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Select Function to Plot</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="linearButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="expButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Exponential</string>
    </property>
   </widget>
   <widget class="QPushButton" name="polyButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="text">
     <string>Polynomial</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
